webpack/stripes-local-federation-plugin.js

```diff
@@ -0,0 +1,42 @@
+// This plugin is used for local development/debugging related to Module Federation of ui-modules.
+// it spawns child processes to start remote ui-modules defined in stripes.config.js if those modules are included
+// within the same workspace. If the module is not within the same workspace, the 'stripes federate' command will have to be
+// executed manually from the directory of that module.
```

Comment:
I don't understand the purpose of this plugin. The whole point of module-federation is that discovery moves from a static list configured at build-time in stripes.config.js to a dynamic list read at run-time from the entitlement API response.